foundations/tying_it_all_together/hire-beware.md

```diff
@@ -0,0 +1,50 @@
+You finished Foundations, congratulations! You may be tempted- like many before you- to consider applying to jobs. But the reality is that you only know the basics of web development and are far from job-ready. It is understandable, really: we don't know what we don't know.
+
+What may amplify this false belief are the job listings that offer paid training, who then contract you out as consultants. While these opportunities are real and not necessarily a scam, they are severely less than ideal. Read on to learn more about these job opportunities.
+
+### Warning Signs and What To Consider
+
+People often ask questions on Discord about companies that offer free, paid training and then contract you out as consultants. Probably the most prominent company that uses this model is Revature. But there are countless other companies that employ this model and- while the details may vary- they will operate similarly. If in doubt, look up the company on Glassdoor, reddit, or any other source to see what you can find out about them. The job postings usually mention that you will be working as a consultant after their free training program. These companies manage to post job listings seemingly everywhere and are enticing because they are among the only job postings that say “entry level,” “junior,” or “no experience required.”
```

Comment:
We should avoid the pattern of using hyphens in place of using commas: 
`his model and- while the details may vary- they wil...` -> `this model, while the details may vary, they will...`

This should be reflected across the writing.